pkg/_popularity/mono_pkg.yaml

```diff
@@ -0,0 +1 @@
+# See https://github.com/dart-lang/mono_repo for details
```

Comment:
Do we need these empty files around?

Reply:
They allow `mono_repo --recursive pub get` to work...